src/Entity/Delivery.php

```diff
@@ -160,9 +164,15 @@ class Delivery extends TaskCollection implements TaskCollectionInterface, Packag
      */
     private $store;
 
+    /**
+     * @var DeliveryPriceInput
+     * @Groups({"delivery_create"})
+     */
+    private $deliveryPriceInput;
```

Comment:
AFAIK there is no need to create a new `DeliveryPriceInput` for this, you can use the existing `ArbitraryPrice`. 
This way in `Appbundle\Action\Delivery\Create`, you don't need to convert the `DeliveryPriceInput` into an `ArbitraryPrice`. Just put the `@var ArbitraryPrice` and/or add type hinting, and API Platform will understand.